waspc/src/Wasp/Generator/WebAppGenerator.hs

```diff
@@ -120,7 +120,7 @@ genPackageJson spec waspDependencies = do
             [ "appName" .= (fst (getApp spec) :: String),
               "depsChunk" .= N.getDependenciesPackageJsonEntry combinedDependencies,
               "devDepsChunk" .= N.getDevDependenciesPackageJsonEntry combinedDependencies,
-              "nodeVersionRange" .= show NodeVersion.nodeVersionRange
+              "nodeVersionRange" .= (">=" <> show NodeVersion.oldestWaspSupportedNodeVersion)
```

Comment:
I am not against `++` per se, but I like using `<>`, same like I like using just `fmap` instead of using `map` for lists. It is more uniform, I don't really care if it is a String or Text or something else, ... .

Reply:
I disagree (especially with the `fmap` part), but you already know that :smile: 

Anyway, you don't need to change it.

Reply:
I believe we both know we disagree on this.